src/sonic-config-engine/config_samples.py

```diff
@@ -39,11 +48,44 @@ def generate_empty_config(data):
     return new_data
 
 def generate_l2_config(data):
+    if 'is_dualtor' in data and data['is_dualtor']:
+        is_dualtor = True
+        data.pop('is_dualtor')
+    else:
+        is_dualtor = False
     data['VLAN'] = {'Vlan1000': {'vlanid': '1000'}}
     data['VLAN_MEMBER'] = {}
-    for port in natsorted(data['PORT']):
+    if is_dualtor:
```

Comment:
I do not see a setting for Loopback2  interface. This will stall linkmgrd.

Reply:
Updated, thanks